components/DragDropTierList.tsx

```diff
@@ -49,7 +49,7 @@ const TierItems = memo<{
                   {...provided.draggableProps}
                   {...provided.dragHandleProps}
                   className={`
-                    m-0.5 rounded-md 
```

Comment:
The spacing produced by this `m-0.5` is preserved by adding `gap-1 m-0.5` to L42